installer/source/installer.c

```diff
@@ -51,9 +51,9 @@ void patch_kernel() {
     // patch ptrace, thanks 2much4u
     *(uint8_t *)(kernbase + 0x0010F879) = 0xEB;
 
-	// Need to find 6.72 offsets.
     // remove all these bullshit checks from ptrace, by golden
-    //memcpy((void *)(kernbase + 0x30DE01), "\xE9\xD0\x00\x00\x00", 5);
+    // updated 6.72 patch ~DeathRGH 06/08/2020
+    memcpy((void *)(kernbase + 0x10FD22), "\xE9\xE2\x02\x00\x00", 5);
```

Comment:
The problem I am seeing with ptrace is that when PT_ATTACH is done, even after sending PT_CONTINUE, the process is not resuming its execution. Even I did this patch but I used `memcpy((void *)(kernbase + 0x10FD22), "\xE9\x75\xFF\xFF\xFF", 5);` but it didn't work. Tried yours too but that didn't solve the issue. No one seems to know what this patch is for? Do you have any idea on that?